$OpenBSD: patch-configure,v 1.29 2013/12/24 08:18:46 ajacoutot Exp $

Our libdb does not follow upstream naming (libdb.so.XX versus
libdb-5.0.so).

--- configure.orig	Mon Dec 23 13:40:08 2013
+++ configure	Tue Dec 24 09:08:12 2013
@@ -22050,9 +22050,9 @@ $as_echo_n "checking for Berkeley DB files in $d... " 
 		        db_libfile="$d/$bdb_loclibdir/libdb$sversion$bdb_libext"
         	        try_headers="db.h db$version/db.h db$sversion/db.h"
 		    else
-		        db_libfilename="libdb-$version$bdb_libext"
-        	        db_lib="-ldb-$version"
-		        db_libfile="$d/$bdb_loclibdir/libdb-$version$bdb_libext"
+		        db_libfilename="libdb$bdb_libext.${LIBdb_VERSION}"
+        	        db_lib="-ldb"
+		        db_libfile="$d/$bdb_loclibdir/libdb$bdb_libext.${LIBdb_VERSION}"
         	        try_headers="db.h db$version/db.h db$sversion/db.h"
     		    fi
 		fi
